src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/NovelAdjacencyReferenceLocations.java

```diff
@@ -57,64 +58,78 @@ protected NovelAdjacencyReferenceLocations(final Kryo kryo, final Input input) {
         this.complication = kryo.readObject(input, BreakpointComplications.class);
     }
 
-    /**
-     * Returns the reference coordinates of the left and right breakpoints implied by this chimeric alignment.
-     * If there is homologous sequence represented in the alignments, it will be assigned to the side of the breakpoint
-     * with higher reference coordinates.
-     */
-    @VisibleForTesting
-    static Tuple2<SimpleInterval, SimpleInterval> leftJustifyBreakpoints(final ChimericAlignment ca,
-                                                                         final BreakpointComplications complication,
-                                                                         final SAMSequenceDictionary referenceDictionary) {
+    private abstract static class BreakpointLeftAligner {
```

Comment:
Don't call these `Aligners` since that's a word with a specific meaning in our field (ie. actual string alignment). How about `ContigToReferenceBreakpointMapping` or something like that?

Reply:
Roger. Renamed from `LeftAligner` to `Inference`